include/silo/config/preprocessing_config.h

```diff
@@ -51,13 +41,7 @@ class PreprocessingConfig {
    std::optional<uint32_t> duckdb_memory_limit_in_g;
    std::optional<std::filesystem::path> lineage_definitions_file;
    std::optional<std::filesystem::path> ndjson_input_filename;
```

Comment:
Can we make this non-optional now?

Reply:
If this were non-optional, it would need a default value. Anyways, I would wait for the config refactor for this. Maybe it will resolve this already